### PR TITLE
Adawy lcd

### PR DESCRIPTION
**Pull request by 3dawy**

### CODE REVIEW

**Comment by YehiaSprints** [comment]
Please complete the tester and submit into the sheet.

**Comment by YehiaSprints** [comment]
No comment from Mohammed Tarek.

**Comment by BassamSharkawy-Sprints** [comment]
Please apply Yahia's/M.Tarek's reviews and merge.

File: ADAWY/TEST/LCD_Test/LCD_TestApp.c [hunk 1]
```diff
@@ -0,0 +1,676 @@
...
+		printf("Please enter test number: ");
+
+		scanf("%d",&au8_TestNumber);
+
+		switch (au8_TestNumber)
+		{
+		case 1:
+			gu8_DIO_Read_Return = E_OK;
+			gu8_DIO_WritePort_Return = E_OK;
+			au8_Result = LCD_DisplayChar('A');
```
Comment (YehiaSprints) [inline ADAWY/TEST/LCD_Test/LCD_TestApp.c:265]: What if the input character is not in the valid ASCII range?

File: ADAWY/TEST/LCD_Test/LCD_TestApp.c [hunk 2]
```diff
@@ -0,0 +1,676 @@
...
+
+		scanf("%d",&au8_TestNumber);
+
+		switch (au8_TestNumber)
+		{
+		case 1:
+			gu8_DIO_Read_Return = E_OK;
+			gu8_DIO_WritePort_Return = E_OK;
+			au8_Result = LCD_DisplayString("AK",2);
+			printf("The expected output is 1  and the actual output is %d", au8_Result);
```
Comment (YehiaSprints) [inline ADAWY/TEST/LCD_Test/LCD_TestApp.c:347]: What did you mean by 1 as the expected output?

File: ADAWY/TEST/LCD_Test/LCD_TestApp.c [hunk 3]
```diff
@@ -0,0 +1,676 @@
...
+		scanf("%d",&au8_TestNumber);
+
+		switch (au8_TestNumber)
+		{
+		case 1:
+			gu8_DIO_Read_Return = E_OK;
+			gu8_DIO_WritePort_Return = E_OK;
+			LCD_init();
+			au8_Result = LCD_DisplayStringRowColumn("AKS",3,0);
+			printf("The expected output is 0  and the actual output is %d", au8_Result);
```
Comment (YehiaSprints) [inline ADAWY/TEST/LCD_Test/LCD_TestApp.c:436]: What did you mean by 0 as the expected output?

File: ADAWY/TEST/LCD_Test/LCD_TestApp.c (hunk 2 above)
Comment (3dawy) [reply ADAWY/TEST/LCD_Test/LCD_TestApp.c:347]: it's an error value

File: ADAWY/TEST/LCD_Test/LCD_TestApp.c (hunk 1 above)
Comment (3dawy) [reply ADAWY/TEST/LCD_Test/LCD_TestApp.c:265]: ok I will handle this case

**Pull request merged**